process/subprocesses/ghsci.py

```diff
@@ -650,12 +650,8 @@ def _region_dictionary_setup(self, folder_path):
             'urban_region',
             data_path,
         )
-        if r['urban_region']['data_dir'].startswith('Not required'):
-            r['urban_query'] = None
-            if r['study_region_boundary']['data'] == 'urban_query':
-                sys.exit(
-                    'Study region has been configured to use the "urban_query" parameter, but urban region data does not appear to have been defined.',
-                )
+        if r['urban_region'] is None:

```

Comment:
Are you sure of the implications for removing this logic?  Seems beyond scope to the earth engine integration?